yew/src/virtual_dom/mod.rs

```diff
@@ -81,6 +81,22 @@ impl Classes {
         self.set.extend(classes_to_add.set);
     }
 
+    /// Pushes the value if the `flag` is `true`.
+    pub fn push_if(&mut self, flag: bool, class: &str) {
```

Comment:
Maybe we should change `fn push(...)` to use `Into<Option<String>>` so that you can pass option wrapped class names?